.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+# Description
+
+Please include a summary of the changes and relevant context. Please also reference any related issue in your repository.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+# Checklist:
+
+- [ ] Does the affected code have corresponding tests?
+- [ ] Are the changes documented, not just with inline documentation, but also with conceptual documentation such as an overview of a new feature, or task-based documentation like a tutorial? Consider if this change should be announced on your project blog.
+- [ ] Does this introduce breaking changes that would require an announcement or bumping the major version?
+- [ ] Does all new files have appropriate license header?
```

Comment:
Nit: I think it should be "do all new files..."?

Reply:
Fixed this. Will merge.